packages/nx/src/command-line/release/meta.ts

```diff
@@ -0,0 +1,66 @@
+import { readNxJson } from '../../config/nx-json';
+import { createProjectGraphAsync } from '../../project-graph/project-graph';
+import { releaseChangelogCLIHandler } from './changelog';
+import { ChangelogOptions, VersionOptions } from './command-object';
+import {
+  NxReleaseConfig,
+  createNxReleaseConfig,
+  handleNxReleaseConfigError,
+} from './config/config';
+import { NxReleaseVersionResult, releaseVersionCLIHandler } from './version';
+
+export async function releaseMeta(
+  args: VersionOptions & ChangelogOptions
+): Promise<NxReleaseVersionResult | number> {
+  const nxReleaseConfig = await readNxReleaseConfig(args);
+
+  const versionResult: NxReleaseVersionResult | 1 =
+    await releaseVersionCLIHandler({
+      ...args,
+      // if enabled, committing and tagging will be handled by the changelog
+      // command, so we should only stage the changes in the version command
+      stageChanges: args.gitCommit || nxReleaseConfig.git?.commit,
```

Comment:
This logic is the only reason I need to parse the nx config below. If we can remove this, then we can leave the config parsing to the inner commands.